homeassistant/components/homematicip_cloud/__init__.py

```diff
@@ -239,6 +254,33 @@
         schema=SCHEMA_SET_ACTIVE_CLIMATE_PROFILE,
     )
 
+    async def _async_dump_hap_config(service):
+        """Service to dump the configuration of a Homematic IP Access Point."""
+        config_path = service.data[ATTR_CONFIG_OUTPUT_PATH] or hass.config.config_dir
+        config_file_prefix = (
+            service.data[ATTR_CONFIG_OUTPUT_FILE_PREFIX] or DEFAULT_CONFIG_FILE_PREFIX
+        )
+        anonymize = service.data[ATTR_ANONYMIZE]
+
+        for hap in hass.data[DOMAIN].values():
+            hap_sgtin = hap.config_entry.title
+
+            if anonymize:
+                hap_sgtin = hap_sgtin[-4:]
+
+            config_file = f"{config_path}/{config_file_prefix}_{hap_sgtin}.json"
```

Comment:
Use pathlib.Path to construct the path. Use f-strings to create the filename.

Reply:
done